docs/6-guidelines-for-regular-operation.md

```diff
@@ -18,45 +18,34 @@ The Guild should transition to regular operations before the end of the initial
 
 ## 6.3 Weighting
 
-Antifragility and non-gameability emerge from simple frameworks. This limits the time spent deciding appropriate categories, the methods for collecting and verifying  as well as the time spent weighting contributors. Additionally, this should limit cases of ambiguity gaming that might come up in complicated weighting schemes. Based on member feedback, we've settled on these guidelines: SQRT((`eligibleMonths` - `monthsOnBreak`) * `timeWeighting`)
+Antifragility and non-gameability emerge from simple frameworks. This limits the time spent deciding appropriate categories, the methods for collecting and verifying as well as the time spent weighting contributors. Additionally, this should limit cases of ambiguity gaming that might come up in complicated weighting schemes. Based on member feedback, we've settled on these guidelines: SQRT((`eligibleMonths` - `monthsOnBreak`) * `timeWeighting`)
 
   - Historic contributions are considered in weightings
   - `timeWeighting` can be either 1.0 for full-time or .5 for part-time contributors
   - Existing contributors get "diluted" as newcomers come in
   - Continuing contributors get additional weight per month they are active. This means historic contributors don't maintain their split weighting if they leave protocol development
   - Anyone who stops contributing is removed from the split via periodic curation reviews
 
-## 6.4 Proposing and Discussing New Members
+## 6.4 Proposing New Members
 
-If current members believe that the set should be expanded to include another contributor, they should propose them well enough in advance of a membership update. This would mean at least a week. The process would look something like this:
+If current members believe that the set should be expanded to include another contributor, they should propose them well enough in advance of a membership update. The process would look something like this:
 
-1. An existing member (akin to an EIP champion) should send this info in the `proposed-members` channel.
+1. An existing member (akin to an EIP champion) should send the info below as a PR to the private [Membership](https://github.com/protocolguild/membership) repo. The work the potential member is being proposed for should already be happening at the date of the proposal - it should not be for prospective work that will start in the future.
```

Comment:
I think this could be further clarified. In my proposal's case, their previous work was already eligible (i.e. clocked up more than 6 months) but they were in a gap at the time of proposal.

If for example they had instead been proposed a month or two ago, then they wouldn't have been currently in a gap, but would have had the same amount of eligible work.

I think I understand the reasoning that "prospective work" may not transpire and therefore shouldn't be added. I guess the proposal that core work will restart requires a similar level of trust as the condition that existing members will notify if they have a significant gap in their core work.